index.js

```diff
@@ -2,8 +2,6 @@ const defaults = require('./core')
 
 module.exports = {
   ...defaults,
-  env: { ...defaults.env, jest: true },
-  plugins: [...defaults.plugins, 'jest'],
```

Comment:
what's this change for?

Reply:
Per the commit message:
> Cleanup some redundant stuff

These two additional elements are already in `defaults`

Reply:
oh... they shouldn't be... BRB